Parse/PFUser.m

```diff
@@ -1113,11 +1113,11 @@ - (BFTask *)fetchAsync:(BFTask *)toAwait {
     }];
 }
 
-- (void)fetch:(NSError **)error {
+- (instancetype)fetch:(NSError **)error {
     if (self.isLazy) {
-        return;
+        return nil;
```

Comment:
Still return `self` here, though...
Returning `nil` would mean that there was an error, but the idea here is basically that the object doesn't need to be fetched, since it's lazy.

Like in `fetchInBackgroundWithBlock:` we have
```objc
- (void)fetchInBackgroundWithBlock:(PFObjectResultBlock)block {
    if (self.isLazy) {
        if (block) {
            block(self, nil);
            return;
        }
    }
    [super fetchInBackgroundWithBlock:^(PFObject *result, NSError *error) {
        if (block) {
            block(result, error);
        }
    }];
}
```